### PR TITLE
Update threeten

### PR DESCRIPTION
**Pull request by adstro**
Update to the latest ThreeTenBP and adjust travisCI config so tests pass.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ language: android
 
 android:
   components:
+    - platform-tools
+    - tools
```
Comment (JakeWharton) [inline .travis.yml]: You should actually flip these. `tools` is needed first to install the latest `platform-tools` (which I'll update separately).

File: .travis.yml [hunk 2]
```diff
@@ -36,3 +38,7 @@ sudo: false
 cache:
   directories:
     - $HOME/.gradle
+
+after_failure:
```
Comment (JakeWharton) [inline .travis.yml]: Delete this. Add
```groovy
lintOptions {
  textReport true
  textOutput 'stdout'
}
```
to the `build.gradle`'s `android` block instead.

**Comment by JakeWharton** [comment]
Thanks for doing this. I was going to update the other PR but this has almost everything I would have done anyway.

**Comment by adstro** [comment]
Changes made, just waiting for Travis build.

You have accepted PRs I have made in other projects, so I get the feeling you pay attention to PRs. I am happy to help update projects I use and are actively maintained. The power of open-source is crowd-sourced changes :)

**Comment by adstro** [comment]
Travis build passed for me, changes based on PR comments have been pushed.

**Comment by adstro** [comment]
Ping...is there anything else that needs to be changed in the PR?...:)

**Comment by JakeWharton** [comment]
no, but i have to figure out what's causing the build failure before moving forward and i haven't had time

**Comment by adstro** [comment]
Fair enough....no pressure, just wanted to make sure it was not something on my end. This build failure *may* be related to this timeout issue -> https://github.com/travis-ci/travis-ci/issues/3110.

FWIW, the build does pass for me in Travis.

**Pull request closed without merging**